docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,330 @@
+
+# Verify the OCM agent is running on the WEC
+kubectl --context <wec-context> get pods -n open-cluster-management-agent
+```
+
+A successfully registered WEC will show:
+- Status: `Available` and `Joined`
+- OCM agent pods running in the `open-cluster-management-agent` namespace
+- A corresponding mailbox namespace in the ITS
+
+## Post-Registration Configuration
+
+### Labeling WECs
+
+After registration, you should label your WECs to make them selectable by BindingPolicies. Labels help categorize WECs based on their characteristics:
+
+```shell
+# Basic labeling
+kubectl --context <its-context> label managedcluster <wec-name> location-group=edge name=<wec-name>
+
+# Geographic labels
+kubectl --context <its-context> label managedcluster <wec-name> region=us-east zone=us-east-1
+
+# Capability labels
+kubectl --context <its-context> label managedcluster <wec-name> gpu=true cpu-architecture=amd64
+
+# Environment labels
+kubectl --context <its-context> label managedcluster <wec-name> environment=production tier=critical
+
+# Custom organizational labels
+kubectl --context <its-context> label managedcluster <wec-name> team=platform business-unit=engineering
+```
+
+### Customization Properties
+
+You can define additional properties for each WEC using ConfigMaps in the `customization-properties` namespace:
+
+```shell
+# Create customization properties
+kubectl --context <its-context> create configmap <wec-name> \
+  -n customization-properties \
+  --from-literal clusterURL=https://my.clusters/1001-abcd \
+  --from-literal datacenter=us-east-1a \
+  --from-literal maxPods=100
+```
+
+These properties can be used for rule-based transformations when workloads are distributed to the WEC.
+
+## Different Deployment Scenarios
+
+### Kind Clusters
+
+For local development with Kind clusters:
+
+```shell
+# Create Kind cluster
+kind create cluster --name cluster1
+kubectl config rename-context kind-cluster1 cluster1
+
+# Register with ITS (note the --force-internal-endpoint-lookup flag)
+clusteradm --context its1 get token | grep '^clusteradm join' | \
+  sed "s/<cluster_name>/cluster1/" | \
+  awk '{print $0 " --context cluster1 --singleton --force-internal-endpoint-lookup"}' | sh
+
+# Accept registration
+clusteradm --context its1 accept --clusters cluster1
+```
+
+### K3d Clusters
+
+For K3d clusters with load balancer:
+
+```shell
+# Create K3d cluster with port mapping
+k3d cluster create -p "9443:443@loadbalancer" cluster1
+kubectl config rename-context k3d-cluster1 cluster1
+
+# Register with ITS
+clusteradm --context its1 get token | grep '^clusteradm join' | \
+  sed "s/<cluster_name>/cluster1/" | \
+  awk '{print $0 " --context cluster1 --singleton --force-internal-endpoint-lookup"}' | sh
+
+# Accept registration
+clusteradm --context its1 accept --clusters cluster1
+```
+
+### OpenShift Clusters
+
+For OpenShift clusters, omit the `--force-internal-endpoint-lookup` flag:
```

Comment:
This is a little incongruous with the previous section (for k3d). Currently, that section repeats all three clumps of commands, even though there is a difference only in the first. Here there is a repetition of just two clumps of commands. And again, there is a difference in only one. I would revise to remove duplication.

Not a blocker.